fcrepo-webapp/pom.xml

```diff
@@ -500,7 +500,7 @@
             <directory>src/main/resources</directory>
             <filtering>true</filtering>
             <includes>
-              <include>**/repo.xml</include>
+              <include>**/fcrepo-config.xml</include>
```

Comment:
I am glad to see this was easier than creating a jetty-console-maven-plugin extension :)